Client/Frontend/Fakespot/Views/FakespotReviewQualityCardView.swift

```diff
@@ -9,9 +9,25 @@ import ComponentLibrary
 
 // MARK: View Model
 final class FakespotReviewQualityCardViewModel {
+    private struct UX {
+        static let labelFontSize: CGFloat = 15
+    }
+
     private let tabManager: TabManager
     let fakespotLearnMoreLink = FakespotUtils.learnMoreUrl
     var dismissViewController: (() -> Void)?
+    var productSitename: String?
+    var highlightsText: NSAttributedString {
+        let normalFont = DefaultDynamicFontHelper.preferredFont(withTextStyle: .subheadline,
+                                                                size: UX.labelFontSize)
+        let boldFont = DefaultDynamicFontHelper.preferredFont(withTextStyle: .subheadline,
+                                                              size: UX.labelFontSize,
+                                                              weight: .semibold)
+
+        let highlightedWebsite = PartnerWebsite(for: productSitename?.lowercased())?.orderWebsites.first ?? "Amazon"
+        let plainText = String.localizedStringWithFormat(String.Shopping.ReviewQualityCardHighlightsLabel, highlightedWebsite)
+        return plainText.attributedText(boldPartsOfString: ["Highlights"], initialFont: normalFont, boldFont: boldFont)
```

Comment:
This will not work for any other language - so this string needs to be localized as well.